src/SurveyDesign.jl

```diff
@@ -49,14 +48,15 @@ struct SurveyDesign <: AbstractSurveyDesign
     popsize::Symbol
     sampsize::Symbol
     strata::Symbol
-    pps::Bool
+    weights::Symbol # Effective weights in case of singlestage approx supported
+    allprobs::Symbol # Right now only singlestage approx supported
+    pps::Bool # TODO functionality
```

Comment:
Since we don't have support for `pps`, I suggest removing it until we do. There's no point in having a field that does nothing. It confuses the user also, making them think that we do have support for it.